### PR TITLE
feat: transport colors

### PR DESCRIPTION
**Pull request by momentiris**
- give transports colors when they arrive in the app and use that color to display them throughout the app.
- use different icon for highlighted transports
- i think this fixes icon rendering on firefox and safari

### CODE REVIEW

File: packages/engine-ui/src/components/Map.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ const Map = ({ data }) => {
   const layers = [
     mapUtils.toIconLayer(
       mapUtils.vehicleIcon(data.vehicles),
-      UIState.highlightTransport
+      UIState.highlightTransport,
+      'vehicle'
```
Comment (lundeeell) [inline packages/engine-ui/src/components/Map.js:37]: I think in discussions that we said we should call it `transport`, right?

Comment (lundeeell) [reply packages/engine-ui/src/components/Map.js:37]: Goes for whole PR

Comment (momentiris) [reply packages/engine-ui/src/components/Map.js]: Should I just swap vehicle for tarnsport throughout the whole application?

File: docker-compose.yml [hunk 2]
```diff
@@ -15,24 +15,24 @@ services:
...
-      - predictivemovement
-  admin-server:
-    build: ./packages/engine-server
-    environment:
-      - AMQP_HOST=amqp://rabbitmq
-    ports:
-      - 4000:4000
-    networks:
-      - predictivemovement
+  # admin-ui:
```
Comment (lundeeell) [inline docker-compose.yml:18]: whoops

File: packages/engine-ui/src/utils/mapUtils.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import palette, { getColor } from './palette'
 import { GeoJsonLayer, IconLayer } from '@deck.gl/layers'
-import transportIcon from '../assets/vehicle.svg'
+import transportIcon1 from '../assets/transport.svg'
```
Comment (lundeeell) [inline packages/engine-ui/src/utils/mapUtils.js:3]: transportIcon**1**??

File: docker-compose.yml (hunk 2 above)
Comment (momentiris) [reply docker-compose.yml:18]: Ops. I was sure i didn't stage this 🤔

**Pull request merged**